pyproject.toml

```diff
@@ -43,7 +43,7 @@ dependencies = [
     "docutils>=0.19,<0.21",
     "restructuredtext-lint>=0.7",
     "stevedore",
-    "tomli; python_version < '3.11'",
```

Comment:
My mistake removing this; I realize now that the code dynamically uses built-in `toml` for Py3.11+